### PR TITLE
Include escalate plugins filter in model.

### PR DESCRIPTION
**Pull request by johnrengelman**
Related PRs:
https://github.com/drone/drone/pull/1405
https://github.com/drone/drone-exec/pull/20

### CODE REVIEW

File: plugin/types.go [hunk 1]
```diff
@@ -22,10 +22,11 @@ type Repo struct {
...
 type System struct {
-	Version string   `json:"version"`
-	Link    string   `json:"link_url"`
-	Plugins []string `json:"plugins"`
-	Globals []string `json:"globals"`
+	Version         string   `json:"version"`
+	Link            string   `json:"link_url"`
+	Plugins         []string `json:"plugins"`
+	Globals         []string `json:"globals"`
+	EscalatePlugins []string `json:"escalate_plugins"`
```
Comment (jackspirou) [inline plugin/types.go]: Should this change also be present in https://github.com/drone/drone-go/blob/master/drone/types.go#L128 ?  

I think the idea is for `drone-go` to eventually replace `drone-plugin-go`.

Comment (johnrengelman) [inline plugin/types.go]: Yup. I'll duplicate it there. I made the change here because this is the package that `drone-exec` is using.

Comment (johnrengelman) [inline plugin/types.go]: Added: https://github.com/drone/drone-go/pull/17

**Comment by tboerger** [comment]
@bradrydzewski what do you think? Close this pr and just use the new client?

**Comment by johnrengelman** [comment]
I can update drone-exec to use the new client if you want.

**Comment by bradrydzewski** [comment]
this is probably going to be an annoying request since it needs to be updated in 4 places but could we call the json tag `privileged_plugins`? No need to change the Go attribute at this time. Normally I wouldn't be such a stickler for naming, however, the plugin json payloads have strict backward compatibility rules so we only get one shot to name things before they are frozen forever. Don't hate me!

**Comment by johnrengelman** [comment]
@bradrydzewski No problem!

I've also created this PR to migrate drone/drone-exec to use drone/drone-go
https://github.com/drone/drone-exec/pull/22

**Comment by johnrengelman** [comment]
Updated everywhere :)

**Pull request merged**